test/topological_codes/test_codes.py

```diff
@@ -64,7 +64,7 @@ def get_noise(p_meas, p_gate):
 class TestCodes(unittest.TestCase):
     """The test class. """
 
-    def test_rep(self, weighted=False):
+    def test_rep(self, weighted=False, xbasis=False):
```

Comment:
Does this actually execute the test with the multiple parameters? Normally we use `ddt` to do a parameter sweep like this. For example: https://github.com/Qiskit/qiskit-ignis/blob/101cdc01bee1be8cba3cc1103e5e146f43bfce96/test/mitigation/test_mitigators.py#L118-L131

Reply:
Ah yes, you are right. There should really be a run of that with `xbasis=True`. I'll take a look on Monday.